### PR TITLE
remove daysSinceLastScan as it is redundant with certifier interval

### LINKED ISSUE #2077: [ingestion/clearlydefined] clearlydefined fails to run when the osv certifier has already run
**Describe the bug**
 clearlydefined fails to run when the osv certifier has already run. This occurs because root package query checks if certifyVulns are already collected and skips the package. This should not happen when the clearly defined certifier is run. 
 
 Question is, should this check be removed? As we want the certifiers to run on OSV and clearlydefined on various intervals, re-adding new vulns or licenses as they are found. The check was only added to prevent the certifier from running again manually and adding duplicate vuln nodes. This, in the long term, is actually what we want to happen in GUAC as we know that the information is up-to-date.

 
**To Reproduce**

1. `go run ./cmd/guacgql --gql-debug --gql-backend=ent`
2. `go run ./cmd/guacone collect files spdx_vuln.json`
3. `go run ./cmd/guacone certifier osv`
4. `go run ./cmd/guacone certifier cd`

You will notice that it completes without adding any licenses.

**Expected behavior**
Add all the license information based on clearly defined.

**GUAC version**
v0.8.0

### PR DESCRIPTION
**Pull request by pxp928**
# Description of the PR

fixes #2077 

# PR Checklist

- [x] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [x] All new changes are covered by tests
- [ ] If GraphQL schema is changed, `make generate` has been run
- [ ] If GraphQL schema is changed, GraphQL client updates/additions have been made
- [ ] If OpenAPI spec is changed, `make generate` has been run
- [ ] If ent schema is changed, `make generate` has been run
- [ ] If `collectsub` protobuf has been changed, `make proto` has been run
- [ ] All CI checks are passing (tests and formatting)
- [ ] All dependent PRs have already been merged

### CODE REVIEW

**Review by jeffmendoza — APPROVED** [review]

**Review by mihaimaruseac — COMMENTED** [review]
I think this makes sense, but leaving just a comment so that others get a chance to look at it too

**Comment by pxp928** [comment]
Looks like I need to fix the e2e test here

**Comment by pxp928** [comment]
should be fixed now

File: internal/testing/e2e/e2e [hunk 1]
```diff
@@ -148,8 +148,6 @@ for command in "${ingestion_commands[@]}"; do
   echo @@@@ Waiting
   sleep 60
 
-  go run ${GUAC_DIR}"/cmd/guacone" certifier osv -p=false
```
Comment (pxp928) [inline internal/testing/e2e/e2e:151]: removed `guacone certifier osv` as `--add-vuln-on-ingest=true` is already being called. This caused duplicate vulns to show up and break the e2e test.

**Review by lumjjb — APPROVED** [review]

**Pull request merged**